src/event.rs

```diff
@@ -44,6 +44,17 @@ use druid_shell::window::{MouseEvent, TimerToken};
 /// [`WidgetPod`]: struct.WidgetPod.html
 #[derive(Debug, Clone)]
 pub enum Event {
+    /// Called on the root widget when the window size changes.
```

Comment:
I could imagine wanting a bunch of events that only go to the root widget; things like application life-cycle, opening files, or doing state restoration. I could imagine something like,

```rust
pub enum RootEvent {
    LocaleChanged(Locale),
    Start,
    WillQuit,
    // ...
    WindowEvent(Event),
}
```